src/arch/amdgpu/vega/pagetable_walker.cc

```diff
@@ -412,11 +464,24 @@ Walker::recvTimingResp(PacketPtr pkt)
 
     DPRINTF(GPUPTWalker, "Got response for %#lx from walker %p -- %#lx\n",
             pkt->getAddr(), senderState->senderWalk, pkt->getLE<uint64_t>());
+    if (enable_pwc && pwcLookup(pkt->getAddr()) == nullptr) //not found, push
+        pwcInsert(pkt->getAddr(), pkt->getLE<uint64_t>());
```

Comment:
Please use curly braces... I know it seems silly, but until you spend hours tracking down a bug because someone else thought they would save themselves 2 characters...